### ISSUE TITLE
Regarding the mechanism

### ISSUE THREAD

**Issue by wzczc** [xmed-lab/TAM#13]
Thanks for your great work! I have achieved great visualization results using it, but I have some questions:

1. First, I noticed that the code uses the logits of the last hidden state at each generating step. I don't quite understand why this can be used to explain the focus during token generation. Also, logit lens seems to be a similar approach—what's the difference between it and logit lens?

2. Second, I used the attention for each token to generate heatmaps, but the results were much worse than yours. What's the difference in interpretability between TAM and attention maps? Which one is more suitable for interpretability?

I am a beginner in MLLM interpretability, so some of my questions might be basic. I hope you don't mind. Thank you. :)
